crates/bevy_ecs/src/system/query.rs

```diff
@@ -386,7 +386,7 @@ impl<'w, 's, Q: QueryData, F: QueryFilter> Query<'w, 's, Q, F> {
     /// For example, `Query<(&mut A, &B, &mut C), With<D>>` will become `Query<(&A, &B, &C), With<D>>`.
```

Comment:
This might be a little easy to misread because the `D` in the doc string is not referring to the `D` type param. Maybe change to `X`, `Y`, `Z`, `W`?

Reply:
Agreed!

Reply:
```suggestion
    /// For example, `Query<(&mut D1, &D2, &mut D3), With<F>>` will become `Query<(&D1, &D2, &D3), With<F>>`.
```